application/pom.xml

```diff
@@ -2,32 +2,38 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <parent>
-        <groupId>eteosf.hexagonal</groupId>
-        <artifactId>hexagonal-architecture-poc</artifactId>
-        <version>0.0.1-SNAPSHOT</version>
-    </parent>
-    <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>application</artifactId>
-    <name>application</name>
-    <description>application</description>
+  <!-- POM Relationships -->
 
-    <dependencies>
-        <dependency>
-            <groupId>eteosf.hexagonal</groupId>
-            <artifactId>domain</artifactId>
-            <version>0.0.1-SNAPSHOT</version>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-context</artifactId>
-            <optional>true</optional>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework</groupId>
-            <artifactId>spring-beans</artifactId>
-        </dependency>
-    </dependencies>
+  <modelVersion>4.0.0</modelVersion>
 
-</project>
\ No newline at end of file
+  <parent>
+    <groupId>eteosf.hexagonal</groupId>
+    <artifactId>hexagonal-architecture-poc</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>application</artifactId>
+
+  <name>application</name>
+  <description>application</description>
+
+  <!-- Dependencies -->
```

Comment:
This section doesn't appear in the reference pom. Dependencies are included in Pom relationship section.